exporter/elasticsearchexporter/config.go

```diff
@@ -351,11 +363,16 @@ func (cfg *Config) MappingMode() MappingMode {
 	return mappingModes[cfg.Mapping.Mode]
 }
 
-func logConfigDeprecationWarnings(cfg *Config, logger *zap.Logger) {
+func handleDeprecatedConfig(cfg *Config, logger *zap.Logger) {
 	if cfg.Mapping.Dedup != nil {
 		logger.Warn("dedup is deprecated, and is always enabled")
 	}
 	if cfg.Mapping.Dedot && cfg.MappingMode() != MappingECS || !cfg.Mapping.Dedot && cfg.MappingMode() == MappingECS {
 		logger.Warn("dedot has been deprecated: in the future, dedotting will always be performed in ECS mode only")
 	}
+	if cfg.Retry.MaxRequests != 0 {
```

Comment:
This means that if user sets both `max_requests` and `max_retries`, the value of `max_requests` will be used, and a warning will be displayed that `max_requests` is deprecated.

Wouldn't it be better to disallow setting both options?